src/standard/http/response/StatusDescription.ts

```diff
@@ -0,0 +1,100 @@
+  IMUsed: StatusDescriptionEnum.IMUsed,
+  MultipleChoices: StatusDescriptionEnum.MultipleChoices,
+  MovedPermanently: StatusDescriptionEnum.MovedPermanently,
+  Found: StatusDescriptionEnum.Found,
+  SeeOther: StatusDescriptionEnum.SeeOther,
+  NotModified: StatusDescriptionEnum.NotModified,
+  UseProxy: StatusDescriptionEnum.UseProxy,
+  TemporaryRedirect: StatusDescriptionEnum.TemporaryRedirect,
+  PermanentRedirect: StatusDescriptionEnum.PermanentRedirect,
+  BadRequest: StatusDescriptionEnum.BadRequest,
+  Unauthorized: StatusDescriptionEnum.Unauthorized,
+  PaymentRequired: StatusDescriptionEnum.PaymentRequired,
+  Forbidden: StatusDescriptionEnum.Forbidden,
+  NotFound: StatusDescriptionEnum.NotFound,
+  MethodNotAllowed: StatusDescriptionEnum.MethodNotAllowed,
+  NotAcceptable: StatusDescriptionEnum.NotAcceptable,
+  ProxyAuthenticationRequired:
+    StatusDescriptionEnum.ProxyAuthenticationRequired,
+  RequestTimeout: StatusDescriptionEnum.RequestTimeout,
+  Conflict: StatusDescriptionEnum.Conflict,
+  Gone: StatusDescriptionEnum.Gone,
+  LengthRequired: StatusDescriptionEnum.LengthRequired,
+  PreconditionFailed: StatusDescriptionEnum.PreconditionFailed,
+  PayloadTooLarge: StatusDescriptionEnum.PayloadTooLarge,
+  URITooLong: StatusDescriptionEnum.URITooLong,
+  UnsupportedMediaType: StatusDescriptionEnum.UnsupportedMediaType,
+  RangeNotSatisfiable: StatusDescriptionEnum.RangeNotSatisfiable,
+  ExpectationFailed: StatusDescriptionEnum.ExpectationFailed,
+  Imateapot: StatusDescriptionEnum.Imateapot,
+  MisdirectedRequest: StatusDescriptionEnum.MisdirectedRequest,
+  UnprocessableEntity: StatusDescriptionEnum.UnprocessableEntity,
+  Locked: StatusDescriptionEnum.Locked,
+  FailedDependency: StatusDescriptionEnum.FailedDependency,
+  TooEarly: StatusDescriptionEnum.TooEarly,
+  UpgradeRequired: StatusDescriptionEnum.UpgradeRequired,
+  PreconditionRequired: StatusDescriptionEnum.PreconditionRequired,
+  TooManyRequests: StatusDescriptionEnum.TooManyRequests,
+  RequestHeaderFieldsTooLarge:
+    StatusDescriptionEnum.RequestHeaderFieldsTooLarge,
+  UnavailableForLegalReasons: StatusDescriptionEnum.UnavailableForLegalReasons,
+  InternalServerError: StatusDescriptionEnum.InternalServerError,
+  NotImplemented: StatusDescriptionEnum.NotImplemented,
+  BadGateway: StatusDescriptionEnum.BadGateway,
+  ServiceUnavailable: StatusDescriptionEnum.ServiceUnavailable,
+  GatewayTimeout: StatusDescriptionEnum.GatewayTimeout,
+  HTTPVersionNotSupported: StatusDescriptionEnum.HTTPVersionNotSupported,
+  VariantAlsoNegotiates: StatusDescriptionEnum.VariantAlsoNegotiates,
+  InsufficientStorage: StatusDescriptionEnum.InsufficientStorage,
+  LoopDetected: StatusDescriptionEnum.LoopDetected,
+  NotExtended: StatusDescriptionEnum.NotExtended,
+  NetworkAuthenticationRequired:
+    StatusDescriptionEnum.NetworkAuthenticationRequired,
+};
+
+// FILE MARKER - PUBLIC API ////////////////////////////////////////////////////
+
+export { StatusDescription };
```

Comment:
I guess it's the same question, the sole purpose is converting from enum to object?

Reply:
i have the same comment about this not making sense to me. see my comment above: https://github.com/drashland/drash/pull/698#discussion_r1312383633

Reply:
refactored this to use the tuple type you showed: [2409f9d](https://github.com/drashland/drash/pull/698/commits/2409f9d02fd28fcb7b56975ceda1a85cfb2be0b1)